src/simd/packed_simd_impl.rs

```diff
@@ -211,6 +211,7 @@ macro_rules! impl_simd_value(
         }
 
         impl Simd<$t> {
+            #[inline]
```

Comment:
Should this be `inline(always)` or just `inline`?

Reply:
Just `inline` is OK.